lib/topology_docker/node.py

```diff
@@ -78,18 +96,25 @@ def __init__(
         self._hostname = hostname
         self._client = Client(version='auto')
 
+        self._container_name = '{identifier}_{pid}_{timestamp}'.format(
+            identifier=identifier, pid=getpid(),
+            timestamp=datetime.now().isoformat().replace(':', '-')
+        )
+        self._shared_dir_mount = shared_dir_mount
+        self._shared_dir = '/tmp/topology/{}'.format(self._container_name)
```

Comment:
In the future, the framework itself may find it necessary to store temporary data, please change ``topology`` to ``toplogy_docker``.